processing/src/main/java/io/druid/segment/filter/AndFilter.java

```diff
@@ -54,9 +54,14 @@ public ImmutableBitmap getBitmapIndex(BitmapIndexSelector selector)
       return filters.get(0).getBitmapIndex(selector);
     }
 
-    List<ImmutableBitmap> bitmaps = Lists.newArrayList();
-    for (int i = 0; i < filters.size(); i++) {
-      bitmaps.add(filters.get(i).getBitmapIndex(selector));
+    final List<ImmutableBitmap> bitmaps = Lists.newArrayListWithCapacity(filters.size());
```

Comment:
I dunno, I think the `Lists` methods have nicer names and overloads.

```java
List<Integer> xs = Lists.newArrayList(filters.size()); // a List<Integer> with one element, filters.size()
List<Filter> xs = Lists.newArrayList(filters); // a List<Filter> that is a copy of "filters"
List<Filter> xs = Lists.newArrayListWithCapacity(filters.size()); // a List<Filter> with filters.size() elements
```

seems pretty nice to me. Especially since the first doesn't have a jdk equivalent, and the second's jdk equivalent only works on Collections, whereas guava works on Iterables and Iterators too.